src/ucp/core/ucp_mm.c

```diff
@@ -507,14 +507,17 @@ static ucs_status_t ucp_memh_register_gva(ucp_context_h context, ucp_mem_h memh,
         memh->md_map |= UCS_BIT(md_index);
     }
 
+    if (context->config.ext.gva_enable == UCS_CONFIG_AUTO) {
```

Comment:
what if context->config.ext.gva_enable is UCS_YES? then UCP_MEMH_FLAG_HAS_GVA flag is not really represent the state of the memh, which is confusing during debug
maybe rename the flag to UCP_MEMH_FLAG_HAS_AUTO_GVA or similar?